index.html

```diff
@@ -195,6 +195,26 @@ <h2 class="union-header white-background">Skills</h2>
       </ul>
       <div class="h-rule"></div>
     </section>
+    <form action="https://formspree.io/f/mdojqvjo" class="contact-form" method="POST">

```

Comment:
According to the Figma design, kindly make sure you stick to the design as much as possible incluing the font, colours, images and spacing (margin/padding).

> ## The Figma Design
>
><img width="307" alt="Screenshot 2023-06-07 at 2 11 37 PM" src="https://github.com/MargaretTechworld/Portfolio/assets/91237525/66d141c8-3488-4ef9-9233-f7238ba762fa">
>
> ## Your Design
>
><img width="372" alt="Screenshot 2023-06-07 at 2 13 02 PM" src="https://github.com/MargaretTechworld/Portfolio/assets/91237525/3025bee5-b557-41b4-8d3f-ab593eb810a8">